doc/changelog/06-Ltac2-language/17575-ltac2in1.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  Ltac2 quotations `ltac2val:(ltac2 tactic)` in Ltac1 producing a Ltac1 value
```

Comment:
```suggestion
  Ltac2 quotations `ltac2val:(ltac2 tactic)` in Ltac1 that produce an Ltac1 value
```

Perhaps define a hyperlink to ltac2.rst:1827 with `.. _XXX:` and use ```:ref:`ltac2val:(ltac2 tactic) <XXX>` ``` here.  (Pick a good name.)